tests/dynamic_data/publications/layer_style/layer_style_test.py

```diff
@@ -60,6 +60,8 @@ def test_style_xml(layer: Publication4Test, params, rest_method):
         })
 
         assert_util.is_publication_valid_and_complete(layer)
-        style = process_client.get_workspace_layer_style(layer.workspace, layer.name)
+        response = process_client.get_workspace_publication(layer.type, layer.workspace, layer.name)
+        uuid = response['uuid']
```

Comment:
If you remove `@staticmethod`, I think you can obtain uuid like this:
```
uuid = this.publ_uuids[layer]
```
, no need to call `process_client.get_workspace_publication`